docs/deployment.md

```diff
@@ -0,0 +1,214 @@
+```bash
+$ git clone https://github.com/synapseweb3/forcerelay
+$ cd forcerelay/tools/ibc-test/contracts/deployment
+$ ./upload.sh connection https://testnet.ckbapp.dev/ <your_ckb_address>
+$ ./upload.sh channel https://testnet.ckbapp.dev/ <your_ckb_address>
+$ ./upload.sh packet https://testnet.ckbapp.dev/ <your_ckb_address>
+$ ./upload.sh sudt https://testnet.ckbapp.dev/ <your_ckb_address>
+```
+
+generated deployment files after that are located in the directory `./migration`, please find them and search transaction hashes in the CKB explorer, `args` of each type script in the first transaction `Output` can be found there.
+
+let's record all of above `args`s as **<CONNECTION_TYPE_ARGS>**, **<CHANNEL_TYPE_ARGS>**, **<PACKET_TYPE_ARGS>**, and **<SUDT_TRANSFER_TYPE_ARGS>**, plus **<SUDT_TRANSFER_TYPE_ID>** which can be found right in the `sudt-transfer` migration file.
+
+## Install Forcerelay/Axon
+fetch CKB related info through Axon rpc and extract metadata cell deployment data, find its `code_hash` and `args` of type script, and record them as **<CLIENT_CODE_HASH>** and **<CLIENT_TYPE_ARGS>**.
+
+to install Forcerelay/Axon:
+```bash
+$ git clone https://github.com/synapseweb3/forcerelay
+$ cd forcerelay
+$ cargo install --path crates/relayer-cli --bin forcerelay
+```
+
+create `config.toml` in the path `~/.forcerelay/`:
+```toml
+[global]
+log_level = 'info'
+
+[mode.clients]
+enabled = true
+refresh = true
+misbehaviour = false
+
+[mode.connections]
+enabled = false
+
+[mode.channels]
+enabled = false
+
+[mode.packets]
+enabled = true
+clear_interval = 100
+clear_on_start = true
+tx_confirmation = false
+auto_register_counterparty_payee = false
+
+[rest]
+enabled = false
+host = '127.0.0.1'
+port = 3000
+
+[telemetry]
+enabled = false
+host = '127.0.0.1'
+port = 3001
+
+[[chains]]
+id = 'ckb4ibc-0'
+ckb_rpc = 'https://testnet.ckbapp.dev/'
+ckb_indexer_rpc = 'https://testnet.ckbapp.dev/'
+key_name = 'relayer_ckb_wallet'
+store_prefix = 'forcerelay'
+client_code_hash = <CLIENT_COCE_HASH>
+connection_type_args = <CONNECTION_TYPE_ARGS>
+channel_type_args = <CHANNEL_TYPE_ARGS>
+packet_type_args = <PACKET_TYPE_ARGS>
+
+[chains.packet_filter]
+policy = 'allowall'
+
+[chains.packet_filter.min_fees]
+[chains.onchain_light_clients.Axon]
+chain_id = 'axon-0'
+client_cell_type_args = <CLIENT_TYPE_ARGS>
+ibc_handler_address = <IBC_HANDLER_ADDRESS>
+
+[[chains]]
+id = 'axon-0'
+websocket_addr = "wss://rpc-alphanet-axon.ckbapp.dev/ws"
+rpc_addr = "https://rpc-alphanet-axon.ckbapp.dev"
+contract_address = <IBC_HANDLER_ADDRESS>
+restore_block_count = 10000
+key_name = "relayer_axon_wallet"
+store_prefix = "forcerelay"
+```
+
+write your keys of Axon and CKB in two different files privately, named **<your_ckb_private_file>** and **<your_axon_private_file>**, and then, import them to the installed Forcerelay/Axon:
+```bash
```

Comment:
Specific cases where the private key may need to be supplemented are for relayer_ckb_wallet and relayer_axon_wallet in the configuration.

Reply:
these can be specific and do not change, they just the name of the wallet, I can leave a comment here as a hint